package.json

```diff
@@ -66,20 +71,24 @@
     "jotai-immer": "0.3.0",
     "key-did-provider-ed25519": "4.0.2",
     "key-did-resolver": "4.0.0",
+    "lucide-react": "^0.471.1",
     "multiformats": "12.0.1",
     "object-hash": "3.0.0",
     "pino": "8.8.0",
     "pkh-did-resolver": "2.0.0",
     "quick-lru": "6.1.1",
+    "react-hook-form": "^7.54.2",
     "reflect-metadata": "0.2.2",
     "rxjs": "7.8.1",
     "siwe": "2.3.2",
     "systemjs-webpack-interop": "2.3.7",
+    "tailwind-merge": "^2.6.0",
+    "tailwindcss-animate": "^1.0.7",
     "uint8arrays": "4.0.4",
     "url-loader": "4.1.1",
     "util": "0.12.5",
     "web3-utils": "1.8.1",
-    "zod": "3.22.3"
+    "zod": "^3.24.1"
```

Comment:
please use only static version for all the dependencies

Reply:
done, and I agree, I used what the Shadcn cli added in the dependency